jade/release-build.yaml

```diff
@@ -2,9 +2,9 @@
 # ROS buildfarm release-build file
 ---
 abi_incompatibility_assumed: true
-jenkins_binary_job_priority: 82
+jenkins_binary_job_priority: 122
```

Comment:
It's a pretty odd system. Valid values are 1-99. Lower values are higher priority.